assets/src/schedule.d.ts

```diff
@@ -97,3 +98,4 @@ export type TimepointsByRouteId = ByRouteId<LoadableTimepoints>
 export type TripId = string
 
 export type ViaVariant = string
+export { RouteType }
```

Comment:
**question:** I may be missing something, but does this just re-export `RouteType` from `./models/stopData`?

Reply:
You're not missing something, that is a stray from when `RouteType` used to live in this file. Will remove!